cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -0,0 +1,133 @@
+
+{# ==========================================================================
+
+   sub_nav()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a mega menu sub menu content markup when given:
+
+   sub_nav_title:   A string used for the sub nav title.
+
+   sub_nav_landing: A string used for the url of the sub nav landing page.
+
+   sub_nav_items:   An array of tuples.
+
+   sub_nav_media:   A string composed of html markup.
+
+   ========================================================================== #}
+
+{% macro _sub_nav(sub_nav_title, sub_nav_landing, sub_nav_items, sub_nav_media) %}
+<div class="o-mega-menu_subcontent" aria-expanded="false">
+    <div class="wrapper">
+        <button class="o-mega-menu_subcontent-btn__back">Back</button>
+        <div class="o-mega-menu_subcontent-menu">
+            <span class="o-mega-menu_subcontent-title">
+                <a class="o-mega-menu_subcontent-link" href="{{ sub_nav_landing }}">
+                    {{ sub_nav_title }}
+                </a>
+            </span>
+            <div class="o-mega-menu_subcontent-lists">
+            {%- for sub_nav_item in sub_nav_items %}
+                <ul class="list
+                           list__unstyled
+                           o-mega-menu_subcontent-list">
+                    {%- for href, id, caption in sub_nav_item %}
+                        <li class="list_item
+                                   o-mega-menu_subcontent-item">
+                            <a class="list_link
+                                      {{ 'list_link__current' if href == request.path else '' }}
+                                      {{ 'u-link__disabled' if href == '' else '' }}
+                                      o-mega-menu_subcontent-link"
+                               {{ 'href=' + href | e if href != '' else '' }}>
```

Comment:
This is omitting `href` if blank. This is a carryover from the old code. That seems correct, right?